src/web/topic/components/TopicWorkspace/MoreActionsDrawer.tsx

```diff
@@ -83,39 +88,14 @@ import {
   useFillNodesWithColor,
 } from "@/web/view/userConfigStore";
 
-const imageWidth = 2560;
-const imageHeight = 1440;
-
-const downloadScreenshot = () => {
-  const nodes = getDisplayNodes();
-
-  // thanks react flow example https://reactflow.dev/examples/misc/download-image
-  const nodesBounds = getRectOfNodes(nodes);
-  const transform = getTransformForBounds(nodesBounds, imageWidth, imageHeight, 0.125, 2);
-  const viewportElement = document.querySelector(".react-flow__viewport");
-  if (!viewportElement) throw new Error("Couldn't find viewport element to take screenshot of");
-
-  toPng(viewportElement as HTMLElement, {
-    backgroundColor: "#fff",
-    width: imageWidth,
-    height: imageHeight,
-    style: {
-      width: imageWidth.toString(),
-      height: imageHeight.toString(),
-      transform: `translate(${transform[0]}px, ${transform[1]}px) scale(${transform[2]})`,
-    },
-  })
-    .then((dataUrl) => {
-      const a = document.createElement("a");
-
-      a.setAttribute("download", "topic.png");
-      a.setAttribute("href", dataUrl);
-      a.click();
-    })
-    .catch((error: unknown) => {
-      throw error;
-    });
-};
+const ScreenshotSchema = z.object({
```

Comment:
```suggestion
const ScreenshotFormSchema = z.object({
```
np: to match the naming pattern I've used for others like this